core/src/test/scala/unit/kafka/server/ListOffsetsRequestTest.scala

```diff
@@ -202,7 +210,17 @@ class ListOffsetsRequestTest extends BaseRequestTest {
 
   @ParameterizedTest
   @ValueSource(strings = Array("zk", "kraft"))
-  def testResponseIncludesLeaderEpoch(quorum: String): Unit = {
+  def testResponseIncludesLeaderEpochWithUnstableAPIs(quorum: String): Unit = {
```

Comment:
Could you please add comments for this test? this comment is a good reference (https://github.com/apache/kafka/pull/16841#discussion_r1715730246)